src/adapter.js

```diff
@@ -355,10 +361,29 @@ function createStartFn (karma, jasmineEnv) {
 
     jasmineEnv = jasmineEnv || window.jasmine.getEnv()
 
-    setOption(jasmineConfig.stopOnFailure, jasmineEnv.throwOnExpectationFailure)
-    setOption(jasmineConfig.failFast, jasmineEnv.stopOnSpecFailure)
-    setOption(jasmineConfig.seed, jasmineEnv.seed)
-    setOption(jasmineConfig.random, jasmineEnv.randomizeTests)
+    if (typeof jasmineEnv.configure === 'function') {
+      // it is actually possible to pass jasmineConfig directly to `jasmine.configure`,
+      // but that would break karma-jasmine's documented use of `stopOnFailure`
+      // which is not a valid jasmine config option
```

Comment:
Is this statement true for both versions of Jasmine?

The purpose of this code is to set jasmine config, so we only need to insure that this works for all 3.x.  If `stopOnFailure` is not a jasmine thing in 3.x we don't need to support passing it.

Reply:
I cannot find `stopOnFailure` in Jasmine's source code going back all the way through `1.x.x`.

Here is the karma-jasmine commit (0b59d697eb), and [diff](https://github.com/karma-runner/karma-jasmine/commit/0b59d697eb2443699f0f6592565fb2d0a3b11794#diff-e52a9a67956bd51d044af4058fb939ceR329), that added `stopOnFailure`. It appears that before this commit, it was not easy to configure jasmine at all.